### DISCUSSION TITLE
Can I add a temperature unit next to the value?

### DISCUSSION THREAD

**Discussion by enusr2k** [FelixdelasPozas/TrayWeather#136] [Q&A] [answered]
Hi! Can I add a temperature unit next to the value? So that it was 10'C instead of just a number.
![2022-10-17_155427](https://user-images.githubusercontent.com/115537081/196160561-752e85f7-a4ca-41cf-93bd-99d64c0ba3f5.png)

**Comment by adem4ik** [comment]
@FelixdelasPozas I guess it is better to split such text into 2 tray icons, i.e. `10` | `°С`. But they should share the same font, font size & color. Otherwise it would be too small text fitted into one tray icon.

Another solution - just `10°`.

**Comment by FelixdelasPozas** [comment]
Hi @enusr2k . Adding ºC or ºF will make the text really small and probably unreadable (specially the fahrenheit solution, as 100ºF is not a very high temp but the text surely will be tiny). The solution @adem4ik proposes will solve the problem but will make use of another tray icon just to show the temp units (something that the user already knows). And I really dislike the notion of a single program using 3 tray spaces. 
I will try the other solution @adem4ik proposes to see how it really looks like. But the idea has always been to try to make the temp text as readable as possible. If it looks right I can always put a checkbox in the Icon configuration tab to enable/disable it.

Regards, 
Félix.

**Comment by enusr2k** [comment]
**FelixdelasPozas**
**adem4ik**
Thanks!

**Comment by FelixdelasPozas** [comment, marked as answer]
Hi @enusr2k and @adem4ik .

I've been trying a few things but did not get good results. Adding only the degree character 'º' adds enough width to the text to make it rather small. With **20º** is not much (in fact the size is almost the same as **-20**), but i've tried **-10º** and is very small. 

![Comparison2](https://user-images.githubusercontent.com/12167134/196553748-ee503e12-167e-461a-b65e-f9ff62b1fc77.png)

 Probably if the user chooses a thin font and removes the text border it will gain enough space to look better. But I prefer to avoid "combinations" like that. Ideally the text should be readable in the small area that is a tray icon space. 

The not-zoomed image looks like this: 
![Comparison](https://user-images.githubusercontent.com/12167134/196554391-c34c4c77-7dd4-4675-be06-61055b19a51a.png)

@enusr2k If you have trouble distinguishing the icon from others (maybe similar icons showing another numbers, like CPU/GPU temperature) using another font/color combination may solve the problem. Adding another character to the temperature text won't really add information (the user already knows the units of the temperature) and will take precious space from the icon space. I think is better to leave like it is right now. 

Regards, 
Félix

**Reply by enusr2k** [reply]
Okay, I got it, thank you!

**Comment by adem4ik** [comment]
@FelixdelasPozas Please, can you show the screenshot with `Stretch font vertically` turned on?)

For me text looks fine with `-24` temperature and `Stretch font vertically` enabled. Here is custom location: lat 65.407118, lon 139.509510 I've used.
![image](https://user-images.githubusercontent.com/4707112/196564808-481bde0c-cabd-441f-8ec0-25bc35fae816.png)

**Comment by FelixdelasPozas** [comment]
@adem4ik During the weekend I'll try to get a few more screenshots, with and without the font stretching.
